stable/airflow/README.md

```diff
@@ -107,6 +107,9 @@ scheduler:
         }
 ```
 
+We expose the `scheduler.refreshConnections` value to refresh connections by
+removing them before adding them when they are automatically being imported.
+
```

Comment:
If we are making `scheduler.refreshConnections` default to `true`, perhaps mention that if a user wants to add a password after the initial deployment, they should set `scheduler.refreshConnections` to `false`

Reply:
added the mention